src/esp/assets/PTexMeshData.h

```diff
@@ -64,12 +62,26 @@ class PTexMeshData : public BaseMesh {
   virtual void uploadBuffersToGPU(bool forceReload = false) override;
   virtual Magnum::GL::Mesh* getMagnumGLMesh(int submeshID) override;
 
+  float exposure() const;
+  void setExposure(const float& val);
```

Comment:
Just `float val`, maybe? The reference doesn't save anything (is actually slower than passing by-value in this case).

Same in the two cases below.

Reply:
Definitely you are right. Good catch!